src/main/java/org/polyfrost/hytils/mixin/CommandLimboMixin_FixCommand.java

```diff
@@ -1,35 +0,0 @@
-/*
- * Hytils Reborn - Hypixel focused Quality of Life mod.
- * Copyright (C) 2020, 2021, 2022, 2023  Polyfrost, Sk1er LLC and contributors
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
- */
-
-package org.polyfrost.hytils.mixin;
-
-import org.spongepowered.asm.mixin.Dynamic;
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Pseudo;
-import org.spongepowered.asm.mixin.injection.Constant;
-import org.spongepowered.asm.mixin.injection.ModifyConstant;
-
-@Pseudo
-@Mixin(targets = "me.semx11.autotip.command.impl.CommandLimbo", remap = false)
-public class CommandLimboMixin_FixCommand {
-    @Dynamic("AutoTip")
-    @ModifyConstant(method = "onCommand", constant = @Constant(stringValue = "/achat \u00a7c"))
```

Comment:
Can you reimplement this mixin (probably have to change the target method) so that autotip’s command is just disabled outright?

Reply:
how would you do this? in my testing you cant change getCommandName or getCommandUsage (those register faster than mixin does? idk it didnt work for me). and im not sure how you would do it by just modifying onCommand. i tried to change it to just UChat.say("/limbo") but that crashed my game. and i dont know how to mixin into the main class where the command is registered because no essential so all missing files for import statements refuse to build

Reply:
Use `@Coerce`

Reply:
didnt work. decided to trying to inject into registerCommands and just ci.cancel, that also doesnt work

Reply:
I talked to current maintainer of autotip and they said they can remove it once hypixel adds functionality to /limbo (repo for autotip is different then The one I sent before)

Reply:
that would be ideal yes